crates/typst/src/layout/grid/lines.rs

```diff
@@ -0,0 +1,749 @@
+use std::{num::NonZeroUsize, sync::Arc};
+
+use crate::foundations::Fold;
+use crate::layout::{Abs, Axes};
+use crate::visualize::Stroke;
+
+use super::layout::CellGrid;
+
+#[cfg(test)]
+use super::layout::{Entry, RowPiece};
```

Comment:
I think those imports should be in the test module if possible, i.e. `use super::super::layout::{Entry, RowPiece}`.